tests/rest/admin/test_registration_tokens.py

```diff
@@ -223,20 +223,13 @@ def test_create_unable_to_generate_token(self) -> None:
         # Create all possible single character tokens
         tokens = []
         for c in string.ascii_letters + string.digits + "._~-":
-            tokens.append(
-                {
-                    "token": c,
-                    "uses_allowed": None,
-                    "pending": 0,
-                    "completed": 0,
-                    "expiry_time": None,
-                }
-            )
+            tokens.append((c, None, 0, 0, None))
         self.get_success(
             self.store.db_pool.simple_insert_many(
                 "registration_tokens",
-                tokens,
-                "create_all_registration_tokens",
+                keys={"token", "uses_allowed", "pending", "completed", "expiry_time"},
```

Comment:
This is a set---should it be a list or tuple (i.e. ordered?)

Reply:
Next thought: why didn't mypy spot this? Because `keys: Collection[str]`. Perhaps `Sequence[str]` is what we want?

Reply:
(This is in draft---sorry if I'm treading on your toes here)

Reply:
Thanks, yeah this wasn't ready for review yet I was still running tests and noticed this and another issue!